src/main/scala/edu/berkeley/ce/rockslicing/JointSet.scala

```diff
@@ -0,0 +1,45 @@
+package edu.berkeley.ce.rockslicing
+
+/**
+  * Represents input joint set - allows for pattern matching to easily and clearly extract joint set fields
+  *
+  * @param parameters Array containing the input data representing the joint set. The inputs in each array
+  *                   are strike, dip, joint spacing, persistence, phi, cohesion and optional stochastic parameters
+  */
+case class JointSet(parameters: Array[Double]) {
+  val strike = parameters(0)
+  val dip = parameters(1)
+  val jointSpacing = parameters(2)
+  val persistence = parameters(3)
+  val phi = parameters(4)
+  val cohesion = parameters(5)
+
+  // Indicates whether joint set is deterministic or stochastic
+  val stochasticFlag = if (parameters.length > 6) {
+    true
+  } else {
+    false
+  }
+
+  // Optional stochastic parameters
```

Comment:
This logic will all go away if we include these as constructor parameters with default values of `0.0`.